web/styles/innov-footer.scss

```diff
@@ -0,0 +1,89 @@
+.footerLinkCustom {
+    color: #7FB135 !important;
+    font-weight: 400 !important;
+}
+
+.footerTextCustom {
+    font-weight: 400 !important;
+}
+
+.githubCustom {
+    color:#E6E6E6;
+    width:40px !important;
```

Comment:
same comment as below - I'm surprised by all the `important!` on our own classes. I wouldn't expect to need important except in being very intentional about parts of njwds to override. can they be removed?